plugins/auth-backend-module-oidc-provider/report.api.md

```diff
@@ -39,7 +39,10 @@ export type OidcAuthResult = {
 export namespace oidcSignInResolvers {
   const emailLocalPartMatchingUserEntityName: SignInResolverFactory<
     unknown,
-    unknown
+    | {
+        allowedDomains?: string[] | undefined;
+      }
+    | undefined
```

Comment:
seems possible to remove as well, similar to what you did for vmware cloud?

Reply:
Hmm, how do you mean? Get rid of the common email local part one altogether?